CMakeLists.txt

```diff
@@ -104,7 +106,7 @@ option(VELOX_ENABLE_S3 "Build S3 Connector" OFF)
 option(VELOX_ENABLE_GCS "Build GCS Connector" OFF)
 option(VELOX_ENABLE_ABFS "Build Abfs Connector" OFF)
 option(VELOX_ENABLE_HDFS "Build Hdfs Connector" OFF)
-option(VELOX_ENABLE_PARQUET "Enable Parquet support" OFF)
+option(VELOX_ENABLE_PARQUET "Enable Parquet support" ON)
```

Comment:
PARQUET is OFF by default as it is optional.
Please enable it in the CI job where required.

Reply:
Changed to turn it on when VELOX_BUILD_TESTING is ON. Thanks.